store/subscription.go

```diff
@@ -195,8 +211,19 @@ func (s *subscription) insert(ctx context.Context, events []*model.Transfer) (er
 		return
 	}
 
+	// Calculate tx fee
+	fees := make(map[gethCommon.Address]*big.Int)
+	// Assume the tx and receipt are in the same order
+	for _, tx := range s.txs {
```

Comment:
Under the premise that `tx and receipt are in the same order`, why not to get value at a specific index of array?
```
for i, tx := range s.txs {
    r := s.receipts[i]
    price, _ := new(big.Int).SetString(tx.GasPrice, 10)
    fees[gethCommon.BytesToAddress(tx.Hash)] = new(big.Int).Mul(price, big.NewInt(int64(r.GasUsed)))
}



```

Reply:
yes, you're right.